src/types/onyx/ReportAction.ts

```diff
@@ -44,7 +50,10 @@ type Person = {
 
 type ReportActionBase = {
     /** The ID of the reportAction. It is the string representation of the a 64-bit integer. */
-    reportActionID?: string;
+    reportActionID: string;
+
+    /** @deprecated Replaced by reportActionID. */
+    sequenceNumber?: number;
```

Comment:
Should we add it here, if it wasn't before and actually is deprecated?

Reply:
@rezkiy37 I added because of [this comment](https://github.com/Expensify/App/blob/main/src/libs/ReportActionsUtils.js#L312). Do you mean is that I need to detail the comment more?

Reply:
Yeah, could we add more details to the comment. IMO, If it is not clear from the first view, need to clarify.